src/core/linux/SDL_dbus.c

```diff
@@ -24,12 +24,20 @@
 
 #ifdef SDL_USE_LIBDBUS
 // we never link directly to libdbus.
-static const char *dbus_library = "libdbus-1.so.3";
+#define SDL_DRIVER_DBUS_DYNAMIC "libdbus-1.so.3"
+static const char *dbus_library = SDL_DRIVER_DBUS_DYNAMIC;
 static SDL_SharedObject *dbus_handle = NULL;
 static char *inhibit_handle = NULL;
 static unsigned int screensaver_cookie = 0;
 static SDL_DBusContext dbus;
 
+SDL_ELF_NOTE_DLOPEN(elfnote_pipewire,
```

Comment:
Copy pasta?

Reply:
```suggestion
SDL_ELF_NOTE_DLOPEN(elfnote_rpc_dbus,
```

Reply:
Using `__LINE__`, as suggested by you, obviates the need for variable names.

Thanks for the review and advice!